verification.py

```
@@ -1,37 +1,49 @@
-    page.wait_for_timeout(500)
-
-    # Take screenshot of the hover state of the drag handle
-    page.screenshot(path="/home/jules/verification/screenshots/verification.png")
-    page.wait_for_timeout(1000)
+    # We are in simulation mode, but the wizard is still on step 0. 
+    # Click Next until we reach the end
+    for _ in range(6):
+        try:
+            page.get_by_role("button", name="Next").click(timeout=1000)
+            page.wait_for_timeout(200)
+        except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)